### DISCUSSION TITLE
GoB v3_5_85

### DISCUSSION THREAD

**Discussion by kromar** [JoseConseco/GoB#335] [General]
Bug fixes:
- fixed renaming issues https://github.com/JoseConseco/GoB/issues/267
- fixed zbrush rescaling issues https://github.com/JoseConseco/GoB/issues/276
- text of import button translating now to other languages https://github.com/JoseConseco/GoB/issues/274
- texture node not created when importing materials https://github.com/JoseConseco/GoB/issues/291
- https://github.com/JoseConseco/GoB/issues/318
- Maps connecting to the wrong nodes https://github.com/JoseConseco/GoB/issues/284 ![image](https://user-images.githubusercontent.com/99124486/152680242-9dfab55f-f974-4ad7-a533-9af85e7832b9.png)
- Export no longer sets render output format to .bmp https://github.com/JoseConseco/GoB/issues/282
- https://github.com/JoseConseco/GoB/issues/311
- https://github.com/JoseConseco/GoB/issues/310
- https://github.com/JoseConseco/GoB/issues/324

Improvements:
- cleanup addon preferences https://github.com/JoseConseco/GoB/issues/303
- polygroups can now be imported as materials
- materials can now be exported as polygroups
- option to specify manual GoZ path https://github.com/JoseConseco/GoB/issues/295
 ![image](https://user-images.githubusercontent.com/1472884/165620819-3f8b1aaf-9ad5-446f-b0fb-4f33e6b8b2d1.png)
- optimized zbrush import to avoid history overflow
- 

<hr /><em>This discussion was created from the release <a href='https://github.com/JoseConseco/GoB/releases/tag/v3_5_85'>GoB v3_5_85</a>.</em>

**Comment by sidney-eliot** [comment]
Another issue I've noticed in this new version is that the default import of polygroups is set to `Polygroups to Face Maps`, which is good. But export of polygroups is for some reason set to `From Vertex Groups` instead of `From Face Maps`

**Reply by kromar** [reply]
i did some quick fixes for the slider limitations and changed some of the defaults you mentioned, including the polygroup export default.
v3_5_86
https://github.com/kromar/blender_GoB/archive/refs/heads/disable-automerge-vertices-by-default.zip
https://github.com/kromar/blender_GoB/tree/disable-automerge-vertices-by-default

**Comment by sidney-eliot** [comment]
Could you please check if polygroup export works, because for some reason polygroups just wont export. On ver v3_5_86. I'm exporting them per face map option, but switching to vertex group doesn't fix the issue. It's importing the polygroups as face maps but nothing or completely wrong faces are selected. (I Restarted both applications)
![image](https://user-images.githubusercontent.com/85735034/181063435-af7da1b3-1bdd-475a-95b1-62ab9714b237.png)
